packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,287 @@
+    model: IssuesListDefModel;
+    coreGroups?: List<Principal>;
+    onSelect?: (selected: Principal, name?: String) => any;
+    coreUsers?: List<UserGroup>;
+}
+
+interface SecurityUserGroupState {
+    coreGroups?: List<Principal>;
+    coreUsers?: List<UserGroup>;
+}
+
+export class BasicPropertiesFields extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+    render() {
+        const { model, onInputChange, onSelect } = this.props;
+        return (
+            <Col xs={12} md={6}>
+                <SectionHeading title="Basic Properties" />
+                <CommentSortDirectionDropDown model={model} onSelect={onSelect} />
+                <SingularItemNameInput model={model} onInputChange={onInputChange} />
+                <PluralItemNameInput model={model} onInputChange={onInputChange} />
+            </Col>
+        );
+    }
+}
+
+export class AssignmentOptions extends React.PureComponent<SecurityUserGroupProps, SecurityUserGroupState> {
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            coreGroups: List<Principal>(),
+            coreUsers: List<UserGroup>()
+        };
+    }
+
+    componentDidMount() {
+        getCoreGroups().then((coreGroupsData: List<Principal>) => {
+            this.setState(() => ({
+                coreGroups: coreGroupsData,
+            }));
+        });
+
+        getCoreUsersInGroups().then((coreUsersData: List<UserGroup>) => {
+            this.setState(() => ({
+                coreUsers: coreUsersData,
+            }));
+        });
+    }
+
+    render() {
+        const { model, onSelect } = this.props;
+        const { coreUsers, coreGroups } = this.state;
+        return (
+            <Col xs={12} md={6}>
+                <SectionHeading title="Assignment Options" />
+                <AssignedToGroupInput
+                    model={model}
+                    coreGroups={coreGroups}
+                    onSelect={onSelect}
+                />
+                <DefaultUserAssignmentInput model={model} coreUsers={coreUsers} onSelect={onSelect} />
+            </Col>
+        );
+    }
+}
+
+export class SingularItemNameInput extends React.PureComponent<IssuesListDefBasicPropertiesInputsProps> {
+    getIssueNameHelpTip() {
+        return ISSUES_LIST_DEF_SINGULAR_PLURAL_TIP;
+    }
+    render() {
+        const { model, onInputChange } = this.props;
+        const value = model.singularItemName === null ? 'Issue' : model.singularItemName;
+        return (
+            <Row className="margin-top">
+                <Col xs={3} lg={2}>
+                    <DomainFieldLabel label="Singular item name" helpTipBody={this.getIssueNameHelpTip} required={false} />
+                </Col>
+
+                <Col xs={9} lg={8}>
+                    <FormControl
+                        id="singularItemName"
+                        type="text"
+                        placeholder="Enter a singular name for this Issue"
```

Comment:
I think you can remove this placeholder prop now that you have a help tip on the label.

Reply:
good point, now removed